buildSrc/src/main/java/baritone/gradle/task/CreateDistTask.java

```diff
@@ -53,12 +53,14 @@ protected void exec() throws Exception {
         }
 
         // Copy build jars to dist/
-        Files.copy(this.artifactApiPath,             api,             REPLACE_EXISTING);
-        Files.copy(this.artifactStandalonePath,      standalone,      REPLACE_EXISTING);
-        Files.copy(this.artifactUnoptimizedPath,     unoptimized,     REPLACE_EXISTING);
+        // TODO: dont copy files that dont exist
+        Files.copy(this.artifactApiPath,         api,         REPLACE_EXISTING);
+        Files.copy(this.artifactStandalonePath,  standalone,  REPLACE_EXISTING);
+        Files.copy(this.artifactUnoptimizedPath, unoptimized, REPLACE_EXISTING);
 
         // Calculate all checksums and format them like "shasum"
-        List<String> shasum = Stream.of(api, standalone, unoptimized)
+        List<String> shasum = Files.walk(getRelativeFile("dist"), 1)
+                .filter(p -> p.toString().endsWith(".jar"))
```

Comment:
not good. my `dist/` folder contains a TON of old builds, but I only want it to print out the jars that it just wrote there this time